Fix: Release Worklfow - using old build backend

```
sr/bin/bash -e /home/runner/work/_temp/ddd9e5c2-dd09-46f9-b437-bc069c914f81.sh
python: can't open file '/home/runner/work/bdew-datetimes/bdew-datetimes/setup.py': [Errno 2] No such file or directory
Error: Process completed with exit code 2.
##[debug]Finishing: Build a binary wheel and a source tarball

```

[Action](https://github.com/mj0nez/bdew-datetimes/actions/runs/4700711922/jobs/8335805441)

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -50,7 +50,6 @@ jobs:
           pip install build twine
       - name: Build a binary wheel and a source tarball
         run: |
-          python -m pip install --upgrade build twine
           python -m build
       - name: Publish distribution 📦 to Test PyPI
         uses: pypa/gh-action-pypi-publish@release/v1

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -47,10 +47,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools setuptools-scm wheel twine
+          pip install build twine
       - name: Build a binary wheel and a source tarball
         run: |
-          python setup.py sdist bdist_wheel
+          python -m pip install --upgrade build twine
+          python -m build
       - name: Publish distribution 📦 to Test PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:

